pkg/new-ui/v1beta1/authzn.go

```diff
@@ -52,11 +54,17 @@ func IsAuthorized(verb, namespace, resource, subresource, name string, schema sc
 	if r.Header.Get(USER_HEADER) == "" {
 		return "", errors.New("user header not present")
 	}
+
 	user := r.Header.Get(USER_HEADER)
 	user = strings.Replace(user, USER_PREFIX, "", 1)
 
+	var groups []string
+	if header := r.Header.Get(GROUPS_HEADER); header != "" {
+		groups = strings.Split(header, ",")
```

Comment:
Uhm. Probably, we should mention using `,` as a separator in the documentation.

@andreyvelich @johnugeorge Where should we notify users about this?

Reply:
I don't think we have any documentation regarding the Katib UI auth. I think, we should create separate page for Katib UI, and move all relative docs there. For example, move this one: https://www.kubeflow.org/docs/components/katib/hyperparameter/#accessing-the-katib-ui.

@apo-ger @kimwnasptd @elenzio9 Do we have any place in the docs where we explain users how to use Kubeflow UIs ?

Reply:
Sounds great.
Also, adding more documentation for the Katib UI auth there would be better. Through I see the section, we don't mention the auth.

Reply:
@tenzen-y do you want me to include in this PR a section in the [accessing-the-katib-ui](https://www.kubeflow.org/docs/components/katib/hyperparameter/#accessing-the-katib-ui) about the auth system ?

Reply:
@axel7083 Let's wait until @apo-ger, @elenzio9 , or @kimwnasptd comment.
They were the main contributors to support authzn in Katib UI.

Reply:
I meant we'd like to work on that at https://github.com/kubeflow/website.